docs/docs/dev_docs/contracts/syntax/slow_updates_tree.md

```diff
@@ -0,0 +1,219 @@
+---
+title: Slow Updates Tree
+---
+
+Slow Updates Tree is a data structure that allows for public data to be accessed in both private and public domains. Read the high level overview in the [concepts section](../../../concepts/foundation/communication/public_private_calls/slow_updates_tree.md).
+
+The Slow Updates Tree works by having a current tree and a pending tree, and replacing the current tree with the pending tree after an epoch has passed. Public functions can read directly from the current tree, and private functions can perform a membership proof that values are part of a commitment to the current state of the tree.
+
+On this page you will learn:
+
+1. About the SlowTree.nr smart contract
+2. The components involved in using the Slow Updates Tree 
+3. How you can integrate it into your own smart contract
+
+Scroll down to view [integration guide](./slow_updates_tree.md#how-to-integrate-a-slow-updates-tree) and [reference](./slow_updates_tree.md#reference).
+
+# Exploring an example integration through a **`TokenBlacklist`** Smart Contract
```

Comment:
Wdyt about putting this section at the end? Feels weird to start with an example integration. I think we should start with describing what it is, then explain how to use it, then end with an example in practice.